Added serialize function to activations.py

Nik-code opened the pull request:
## PR Description
- Modified file: https://github.com/unifyai/ivy/blob/main/ivy/functional/frontends/tensorflow/keras/activations.py
- Added a serialize function to convert activation functions back into their string identifiers.
- Ensured compatibility with supported TensorFlow versions and data types.
## Related Issue
Closes https://github.com/unifyai/ivy/issues/1569

Nik-code commented:
@paulaehab Hey, can you please review my PR. Thank you.

paulaehab commented:
Hello @Nik-code  first of all thanks for contributing but could you please add test for the fucntion you will find more details in this link : (https://unify.ai/docs/ivy/overview/deep_dive/ivy_frontends_tests.html)

Nik-code commented:
Hey @paulaehab , I have added the relevant tests for the serialize function in the _test_activation.py_ file. I've added them with compliance to the documentation you mentioned [here](https://unify.ai/docs/ivy/overview/deep_dive/ivy_frontends_tests.html), please review the new commit for the changes, thanks.

abdul-rehman18 commented:
https://github.com/unifyai/ivy/issues/25772

paulaehab commented on ivy/functional/frontends/tensorflow/keras/activations.py:
```diff
@@ -111,6 +111,44 @@ def selu(x):
+    {"2.13.0 and below": ("float16", "float32", "float64")},
+    "tensorflow",
+)
+def serialize(func, custom_objects=None):
```
we need to adhere the Tensorflow frontend names for arguments it was in documentation (activation, use_legacy_format=False) you could have a look here (https://www.tensorflow.org/api_docs/python/tf/keras/activations/serialize )

abdul-rehman18 commented:
https://github.com/unifyai/ivy/pull/25790

@paulaehab 
kindly review it

paulaehab commented:
Hey @Nik-code  could you please follow the instruction of this page https://unify.ai/docs/ivy/overview/contributing/the_basics.html#todo-list-issues to add issue related to serialize function in https://github.com/unifyai/ivy/issues/1569

abdul-rehman18 commented:
@paulaehab 
I also made a pr

paulaehab commented:
> #25790
> 
> @paulaehab kindly review it

Hello @abdul-rehman18 I am very sorry @Nik-code  had already made the function and he just forget to add issue with function name , also it was my fault that I didn't get it I am sorry for that could you pick another function and again sorry for this mistake

abdul-rehman18 commented:
no problem sir @paulaehab 
Thanks

paulaehab commented:
> no problem sir @paulaehab Thanks
Thanks for your understanding and again sorry for this mistake if you make another PR feel free to ask review for me

Nik-code commented:
Hey @paulaehab ,  thank you for the suggestions. I'll make sure to make the requested changes soon and post the commit 😊

Nik-code commented:
Close #25851

Nik-code commented:
Hey @paulaehab I have made a new commit changing the function description according to the TensorFlow documentation, Also I have correctly created a sub-task which I mentioned above that is now referenced in #1569

paulaehab commented:
Hey @Nik-code  thanks for contributing look great for me 😄
